pkgs/misc/pywal/default.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
remove `pkgs` here and add `imagemagick`, `feh` and `python3Packages` explictly.